shift-shift.go

```diff
@@ -242,9 +249,17 @@ func listenKeyboards(
 				case 1: // key down
 					switch ev.Code {
 					case keyFirst:
-						groupFirst = true
+						if dblKeystroke {
+							t, groupFirst = checkTimeout(t, dblKeyTimeout)
+						} else {
```

Comment:
There is no need `else` here. Because `groupFirst` already set to `true` above (at line 245) so the assignment in else became redundant.

Reply:
There's no redundancy: `groupFirst` and `groupSecond` are set only once either by `if ` or by `else`, you propose to set it twice in case of `dblKeystroke`. Although the compiler should be able to iron this into the same code.

Reply:
Yes it is my fault, your code is right. It is all ok then.